modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/resources/META-INF/resources/edit_configuration.jsp

```diff
@@ -55,6 +57,12 @@ renderResponse.setTitle(configurationModelName);
 		<aui:input name="pid" type="hidden" value="<%= configurationModel.getID() %>" />
 
 		<div class="lfr-ddm-container" id="lfr-ddm-container">
+			<c:if test="<%= !Validator.isBlank(configurationModelDescription) %>">
+				<aui:alert closeable="<%= false %>" type="default">
+					<%= configurationModelDescription %>
+				</aui:alert>
```

Comment:
As agreed with Víctor this should not be an alert but a regular <p> right above the form HTML (behind the pre-existing alert)